### PR TITLE
fix: Handle empty author object in article comment templates 

### PR DESCRIPTION
**Pull request by aronshamash**
- sets a default `author` object in article comment templates -  `main` and `magazine`
- Fixes 500 errors in render when view an article with this template as no author object is being currently returned.

### CODE REVIEW

File: packages/article-magazine-comment/src/article-magazine-comment.web.js [hunk 1]
```diff
@@ -29,10 +29,12 @@ class ArticlePage extends Component {
       standfirst
     } = article;
 
+    const authorImage = author && author.image ? author.image : null;
```
Comment (colincclark) [inline packages/article-magazine-comment/src/article-magazine-comment.web.js]: you could have destructed this with a default on line 21

Comment (nathggns) [reply packages/article-magazine-comment/src/article-magazine-comment.web.js]: I think that would be less clean tbh. Destructing multiple levels at once becomes very hard to follow very quickly.

Comment (aronshamash) [reply packages/article-magazine-comment/src/article-magazine-comment.web.js]: I think I prefer the destructed way too. Less logic.
